### ISSUE TITLE
Set up test domain with Google Domains

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#1191]
**TL;DR**: Buy and use a test domain to make sure we have the correct information to set up a domain.  

**Narrative**
I am not sure of the exact IP Address, MX records, etc that we have set up for our current MIT domain.  We'll need to get that information and use it set up another throw away domain.  This will allow us to test using a second domain before the childrenhelpingscience transfer deadline. 

**Implementation Notes**
To start off, we'll need to make sure we get the resource information for our current domain from MIT.  

Next, we're going to need to log into our google domains accound, buy a domain, and successfully set it up. 

1. Log into [Google Domains](https://domains.google.com).
2. Click `Get a domain` from the left menu.
3. Type `lookit-mit.com` into the search field and press enter. <img width="859" alt="Screenshot 2023-04-25 at 10 43 23 AM" src="https://user-images.githubusercontent.com/44074998/234313779-70aa9da6-e28a-496b-b8ba-f068d6d99472.png">
4. Click on the shopping cart icon to add domain the your cart and click the cart icon at the top to check out.  
<img width="847" alt="Screenshot 2023-04-25 at 10 44 38 AM" src="https://user-images.githubusercontent.com/44074998/234314498-f7c142fe-1bd2-4e5e-91d2-edc96a98a4b5.png">

Now that we own the domain, we'll have to create the [resource record ](https://support.google.com/domains/answer/3290350?sjid=519076765189700467-NA) with information we get from MIT about our current domain.

**Comment by okaycj** [comment]
This has been completed.

**Comment by okaycj** [comment]
This has been completed.
